src/transformers/tokenization_bert.py

```diff
@@ -550,14 +550,19 @@ def __init__(
         cls_token="[CLS]",
         mask_token="[MASK]",
         tokenize_chinese_chars=True,
-        max_length=None,
-        pad_to_max_length=False,
-        stride=0,
-        truncation_strategy="longest_first",
         add_special_tokens=True,
         **kwargs
     ):
         super().__init__(
+            BertWordPieceTokenizer(
+                vocab_file,
```

Comment:
Can you name the arguments (e.g. `return_tensors=return_tensors`). It's is more robust to later API changes and better self-documented even though it's a bit more cumbersome to write.